my_class.rb

```diff
@@ -0,0 +1,10 @@
+require './enumerables'
```

Comment:
- [ ] I suggest that you use `require_relative` to import `enumerable` because it's a ruby convention, `Require` should be used for external files, like gems